torch/csrc/jit/register_special_ops.cpp

```diff
@@ -24,6 +26,35 @@ RegisterOperators reg({
         [](Stack& stack) {
           return 0;
         }),
+    Operator(
+        "aten::format(str self, *str args) -> Tensor",
```

Comment:
How does the test case pass if you expect string varargs? Where do we insert implicit casts?

Reply:
It's happening in [`compiler.cpp`](https://github.com/pytorch/pytorch/pull/13157/files#diff-c0a25c14600294e812cfbdd9ac3fcae2R487).

Reply:
So anything becomes implicitly convertible to a string? That's not even the semantics in Python

Reply:
Latest update just ignores types for any remaining varargs, but that's not great either. #13278 adds `str` casts for `IValue`s so then `format` can be typed to take only strings and require explicit casts.